### ISSUE TITLE
using-version-control-in-studio-pro - consistent use of terminology please

### ISSUE THREAD

**Issue by Chris-Karona** [mendix/docs#7309]
Please use the form below, leaving the prefilled data to help us. Thank you.

Page link: [using-version-control-in-studio-pro](https://docs.mendix.com/refguide/using-version-control-in-studio-pro/)

Document link: [_index.md](https://github.com/mendix/docs/blob/development/content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md)

## My Issue/Suggestion

Early in this page it is established that there is a working copy (also called Local Model), a Local Repository and a remote respository (also called Team Server).  The terms are not used consistently through the page, for example: 
"4.4 Pulling
Pulling retrieves the latest changes from the repository."   - which repository.  

The reader is constantly having to interpret the author's intention, which makes it very hard to follow what is already a complex explanation.  Using terms consistently would make it more readable. 

Thank you.

**Comment by MariaShaposhnikova** [comment]
Hi @Chris-Karona ,

Thanks a lot for your feedback!
I went through the document and defined which repository we mean, remote or local. I also tried making the terminology more consistent and straight-forward. You can see all the changes in this PR: https://github.com/mendix/docs/pull/7696

I hope it helps to make the information easier to read and understand.

Best regards,
Maria
